test/mitmproxy/test_command.py

```diff
@@ -309,6 +310,22 @@ def empty(self) -> None:
         pass
 
 
+def test_collect_commands():
```

Comment:
This test is much more implementation-specific than it should be. Mocking hasattr is wrong, it's not too hard to come up with an addon that raises when hasattr() is invoked on it.

Reply:
Thanks for the feedback, I will make the necessary changes ASAP.